page_analyzer/utils/url.py

```diff
@@ -28,3 +29,19 @@ def get_url_status_code(url: str) -> int | None:
         print(e)
         return None
     return r.status_code
+
+
+def parse_page_data(url: str) -> Dict[str, str]:
```

Comment:
I have a feeling that it makes sense to unify it with `get_url_status_code` to avoid firing `GET` request twice
unified function should have a different name